particle/src/main/res/values/attrs.xml

```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <declare-styleable name="ParticleView">
+        <attr name="particlesSpeed" format="enum">
```

Comment:
Why do we use enum? We can use integer value with better customization options i think.

Reply:
modularized code is more user friendly